docs/src/intro.md

```diff
@@ -38,7 +38,7 @@ There are three styles of "trace" in yk:
   * When a hot loop in a program is detected, the actions of the interpreter
     are recorded to make an *AOT IR trace*.
 
-  * The execution trace and AOT IR are then compiled into JIT IR to make a *JIT
-    IR trace*.
+  * The AOT IR trace is combined with AOT IR and then compiled into JIT IR to
```

Comment:
Hmm, this sounds a bit weird. `AOT IR trace` kind of implies that we have already combined the "execution trace" with the AOT IR.

Reply:
It's not perfect terminology. If we can think of a big improvement we can change, but it might be the case that the obvious choices suck equally...